src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -28,6 +28,9 @@
     <ProjectJson>$(MsBuildThisFileDirectory)windows\project.json</ProjectJson>
     <ProjectLockJson>$(MsBuildThisFileDirectory)windows\project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(IsNETNativeLibrary)' != 'true' AND '$(TargetsWindows)' == 'true'">
+    <DefineConstants>>$(DefineConstants);SUPPORTS_WINDOWSIDENTITY</DefineConstants>
+  </PropertyGroup>
```

Comment:
I see FEATURE as our feature, whereas SUPPORTS I see as meaning the platform we're building on supports this dependency.